.gitignore

```diff
@@ -62,6 +62,6 @@ _deps
 lib/*
 !lib/.gitkeep
 .clang-format
-.idea/
 .cmake/
+.idea/
```

Comment:
why not xd, genre move de place c'est cool

Reply:
mdr c un bail de conflit ça